llm/python/README.md

```diff
@@ -0,0 +1,113 @@
+# Benchmarking script for large models
+
+This script provides unified approach to estimate performance for LLM. It is based on pipelines provided by Optimum-Intel and allows to estimate performance for pytorch and openvino models using almost the same code and precollected models.
+
+## Usage
+(1) Need to start the virtual environment of python
+``` bash
+python3 -m venv python-env
+source python-env/bin/activate
+pip install update --upgrade
+```
+(2) Convert model to IRs
+## Conversion script for preparing benchmarking models
+
+`convert.py` allow to reproduce IRs stored on shared drive.
+
+Prerequisites:
+install conversion dependencies using requirements.txt
+
+```bash
+pip install -r requirements.txt
```

Comment:
probably if we have common requirements now for both convert and benchmarking script, dependencies install should be after creating env

Reply:
updated